umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -25,7 +25,9 @@
 
 (defmethod apply-umm-list-update :add-to-existing
   [update-type umm update-field update-value find-value]
-  (update-in umm update-field #(conj % update-value)))
+  (if (vector? update-value)
```

Comment:
Whats the reason to use vector? instead of sequential?

Reply:
Changed to sequential? to cover the list case, just in case.